demisto_sdk/tests/integration_tests/validate_integration_test.py

```diff
@@ -0,0 +1,28 @@
+from os.path import join
+from subprocess import PIPE, run
+
+from demisto_sdk.commands.common.git_tools import git_path
+
+DEMISTO_SDK_PATH = join(git_path(), "demisto_sdk")
+MAIN_MODULE_PATH = join(DEMISTO_SDK_PATH, "__main__.py")
+PYTHON_CMD = "python"
+VALIDATE_CMD = "validate"
+PACK_INTEGRATION_PATH = "Packs/FeedAzure/Integrations/FeedAzure/FeedAzure.yml"
+
+
+def test_integration_validate():
+    results = run(
+        [PYTHON_CMD, MAIN_MODULE_PATH, VALIDATE_CMD, "-p", PACK_INTEGRATION_PATH],
+        stderr=PIPE,
+        stdout=PIPE,
+        encoding='utf-8',
+        cwd=join(DEMISTO_SDK_PATH, "tests", "test_files", "content_repo_example")
+    )
+    stdout = results.stdout
```

Comment:
Also check status code is 1 if you expect a failure

Reply:
done